spark/src/test/scala/org/apache/spark/sql/delta/DeltaProtocolVersionSuite.scala

```diff
@@ -591,12 +581,11 @@ trait DeltaProtocolVersionSuiteBase extends QueryTest
       requiredWriterVersion: Int): String = {
     // When testing flag is enabled Action.supportedReaderVersionNumbers includes 0.
     "[DELTA_INVALID_PROTOCOL_VERSION] " +
-      "Delta protocol version is not supported by this version of Delta Lake: " +
-      "table \"" + tableNameOrPath + "\" requires " +
+      "Unsupported Delta protocol version: table \"" + tableNameOrPath + "\" requires " +
       s"reader version $requiredReaderVersion and " +
       s"writer version $requiredWriterVersion, " +
-      "client supports reader versions 0, 1, 2, 3 and writer versions 0, 1, 2, 3, 4, 5, 7. " +
-      "Please upgrade to a newer release."
+      "but Delta Lake \"" + io.delta.VERSION + "\" supports reader versions 0, 1, 2, 3 " +
```

Comment:
Doesn't really matter, but out of curiosity why not:
```suggestion
      s"but Delta Lake \"${io.delta.VERSION}\" supports reader versions 0, 1, 2, 3 " +
```
(looks like there are other examples as well)

Reply:
I was having issues with interpolated string and quotes. Just found that I can use triple quotes .